### PR TITLE
Error framing should use tracing instead of original_message_id

### PR DESCRIPTION
**Pull request by breerly**
Will fix tests once I know the direction is sane.

### CODE REVIEW

File: python/tchannel/messages/error.py [hunk 1]
```diff
@@ -53,6 +54,6 @@ def error_name(self):
 error_rw = rw.instance(
     ErrorMessage,
     ('code', error_code_rw),                            # code:1
-    ('original_message_id', rw.number(4)),              # id:4
+    ('tracing', common.tracing_rw),                     # tracing:24
```
Comment (breerly) [inline python/tchannel/messages/error.py]: I think this should be tracing:25? @abhinav is that right?

Comment (abhinav) [inline python/tchannel/messages/error.py]: I think tracing_rw only reads 24 bytes for the trace IDs right now. It needs to be modified to read the traceflags and explicit traceflags parsing needs to be removed from everywhere else.

Either that or this needs to read another byte for the traceflags.

Comment (junchaowu) [inline python/tchannel/messages/error.py]: in the common.py, we have tracing_rw = rw.instance(
    Tracing,
    ("span_id", rw.number(8)),      # span_id:8
    ("parent_id", rw.number(8)),    # parent_id:8
    ("trace_id", rw.number(8)),     # trace_id:8
    ("traceflags", rw.number(1)),   # traceflags:1
)

Comment (abhinav) [inline python/tchannel/messages/error.py]: Oh cool. Didn't realize that was fixed.

**Comment by breerly** [comment]
:shipit:

**Pull request merged**